src/sentry/utils/email.py

```diff
@@ -350,16 +351,34 @@ def send(self, to=None, bcc=None, fail_silently=False):
 
     def send_async(self, to=None, bcc=None):
         from sentry.tasks.email import send_email
+        # from sentry/emails/send-invite.txt to send-invite
         messages = self.get_built_messages(to, bcc=bcc)
         for message in messages:
-            safe_execute(send_email.delay, message=message,
-                         _with_transaction=False)
+            safe_execute(
+                send_email.delay,
+                message=message,
+                _with_transaction=False,
+            )
+            logger.info(
```

Comment:
For these tight loops, it's probably worth minimizing the logging overhead by using a `partial` here.

Something like:

```python
log_mail_queued = partial(
    logger.info,
    name='sentry.mail',
    event='mail.queued',
    type=self.type,
)
for message in messages:
    log_mail_queued(to=to, mesage_id=message.extra_headers['Message-Id'])
```